### DISCUSSION TITLE
Daily Standup (Thursday, June 2nd)

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [CarlosCRG19/ijalti-peers#84] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x 
```

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Some advances on the search form

**What are your priorities for the next 24 hours?**:
 - Redefine the entire use case

**Blockers**:
 - npm install on server not working

**Shoutouts**:
 - @DavBot02 for being the cornersotne of the front

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Talked with @CristianCazares 

**What are your priorities for the next 24 hours?**:
 - Define next tasks

**Blockers**:
 - None

**Shoutouts**:
 - @CristianCazares

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Work a ton of signup views and context

**What are your priorities for the next 24 hours?**:
 - Continue with signup

**Blockers**:
 - Time

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  Implemented filtering  aspirant by skills feature

**What are your priorities for the next 24 hours?**:
 -  Don't know

**Blockers**:
 - None

**Shoutouts**:
 - @everyone for being working hard

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Added skills relations to aspirant and job offer entities 
 - Opened a PR containing aspirant search by skillset with @BrandonMagana 
 - Solved a middleware issue

**What are your priorities for the next 24 hours?**:
 - Not sure

**Blockers**:
 - Don't know what to do

**Shoutouts**:
 - @CarlosCRG19 @DavBot02 for their amazing job in the frontend and @BrandonMagana for that awesome psql query
 -
